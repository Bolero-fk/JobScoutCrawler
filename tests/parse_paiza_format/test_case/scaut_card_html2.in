<div class="MessageCardHeader"><img class="ScoutImage" src="sample2.jpg" alt="スカウト一覧画像"></div><div class="MessageCardBody"><div class="MessageCardTitle"><div class="MessageCardTitleName">ABC商事株式会社</div><div class="MessageUnReadPoint" style="display: none;"></div></div><div class="MessageCardText">テスト説明2テスト説明2テスト説明2テスト説明2</div><ul class="MessageCardStatus"><li class="MessageCardStatusItem"><i class="fas fa-yen-sign"></i>  250万 〜 600万円</li><li class="MessageCardStatusItem"><i class="fas fa-map-marker-alt"></i>  大阪府</li><li class="MessageCardStatusItemSub"><i class="fas fa-code"></i> Java</li></ul><div class="MessageCardDate"><span style="margin-right: 0.25rem;">2023/02/12</span><span style="color: rgb(46, 173, 67);">残り22日</span></div></div>