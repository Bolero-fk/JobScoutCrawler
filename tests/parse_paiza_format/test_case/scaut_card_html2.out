{"company_name": "ABC商事株式会社", "description": "テスト説明2テスト説明2テスト説明2テスト説明2", "min_salary": 250, "max_salary": 600, "location": "大阪府", "using_lang": "Java", "recieve_date": "2023/02/12", "remaining_days": 22}